src/view/seed-verify.js

```diff
@@ -46,16 +46,23 @@ const SeedVerifyView = ({ store }) => (
       <Card style={styles.card}>
         <FormSubText>{store.seedVerifyCopy}</FormSubText>
         {store.seedVerifyIndexes.map((seedIndex, i) => (
-          <SeedEntry seedIndex={seedIndex} key={i} />
+          <SeedEntry
+            seedIndex={seedIndex}
+            value={store.wallet.seedVerify[i]}
+            onChangeText={word => wallet.setSeedVerify({ word, index: i })}
+            key={i}
+          />
         ))}
       </Card>
-      <GlasButton onPress={() => {}}>Next</GlasButton>
+      <GlasButton onPress={() => wallet.checkSeed()}>Next</GlasButton>
```

Comment:
I was thinking of adding an issue for the `Return` key being equivalent to pressing `Next` (applies to multiple screens)